### PR TITLE
Test for displaying a BO asset with limited permissions

### PR DESCRIPTION
**Pull request by romcheg**

### CODE REVIEW

File: src/ralph/back_office/tests/test_models.py [hunk 1]
```diff
@@ -468,3 +470,39 @@ def test_requester_must_be_specified(self):
...
+        permissions = Permission.objects.exclude(
+            codename__in=[
+                'view_backofficeasset_hostname_field',
+                'view_backofficeasset_service_env_field',
+                'change_backofficeasset_hostname_field',
+                'change_backofficeasset_service_env_field',
+            ]
+        ).all()
+
+        for p in permissions:
```
Comment (ar4s) [inline src/ralph/back_office/tests/test_models.py]: try ``user.user_permissions.add(*permissions)``

File: src/ralph/back_office/tests/test_models.py [hunk 2]
```diff
@@ -468,3 +470,39 @@ def test_requester_must_be_specified(self):
...
+        user = UserFactory()
+
+        user.is_superuse = False
+        user.is_staff = False
+        user.set_password(passwd)
+
+        user.save()
+
+        # Grant all permissions to the user
+        permissions = Permission.objects.exclude(
```
Comment (ar4s) [inline src/ralph/back_office/tests/test_models.py:488]: please split the test into two separated tests: first for ``hostname``, second for ``service_env`` and cover a conditions in this block of code: https://github.com/allegro/ralph/blob/ng/src/ralph/back_office/admin.py#L71-L79

File: src/ralph/back_office/tests/test_models.py [hunk 3]
```diff
@@ -468,3 +470,39 @@ def test_requester_must_be_specified(self):
...
+
+        self.login_as_user(user=user, password=passwd)
+
+        url = reverse(
+            'admin:back_office_backofficeasset_change',
+            args=(asset.pk,)
+        )
+        resp = self.client.get(url)
+
+        self.assertEqual(resp.status_code, 200)
```
Comment (ar4s) [inline src/ralph/back_office/tests/test_models.py]: add ``self.assertTrue(resp.context['form']['service_env'])``

**Review by ar4s — CHANGES_REQUESTED** [review]

File: src/ralph/back_office/tests/test_models.py (hunk 2 above)
Comment (romcheg) [reply src/ralph/back_office/tests/test_models.py:488]: Makes sense.

File: src/ralph/back_office/tests/test_models.py (hunk 1 above)
Comment (romcheg) [reply src/ralph/back_office/tests/test_models.py]: It worked.

File: src/ralph/back_office/tests/test_models.py (hunk 3 above)
Comment (romcheg) [reply src/ralph/back_office/tests/test_models.py]: @ar4s Please take a look at the new patch set.

**Review by ar4s — APPROVED** [review]

**Pull request merged**